src/test/integration/context.js

```diff
@@ -0,0 +1,104 @@
+import describe from 'tape';
```

Comment:
Why not use riteway?

Reply:
that was an old test file, just moved the file

Reply:
I used `git mv` but it must have lost the connection.